File: DevExpress-Demo/VB/MapDemo.Wpf/MapDemoModuleControl.vb
Imports DevExpress.Xpf.DemoBase
Imports DevExpress.Xpf.Map

Namespace MapDemo

    Public Class MapDemoModule
        Inherits DemoModule

        Protected Overrides Sub Hide()
            For Each map As MapControl In FindMap(Me)
                map.HideToolTip()
            Next

            MyBase.Hide()
        End Sub
    End Class
End Namespace

File: DevExpress-Demo/CS/MapDemo.Wpf/MapDemoModuleControl.cs
using DevExpress.Xpf.DemoBase;
using DevExpress.Xpf.Map;

namespace MapDemo {
    public class MapDemoModule : DemoModule {
        protected override void Hide() {
            foreach(MapControl map in DemoUtils.FindMap(this))
                map.HideToolTip();
            base.Hide();
        }
    }
}
